### PR TITLE
Adding RemoveExistingOnUpgrade and SkipDownloads functionality.

### PR DESCRIPTION
**Pull request by rajkumar42**
Also skips the downloads if the latest version is present.

### CODE REVIEW

**Comment by rajkumar42** [comment]
@chuckries @gregg-miskelly @caslan @wesrupert

File: scripts/GetClrDbg.ps1 [hunk 1]
```diff
@@ -14,6 +14,12 @@ Specifies the .NET Runtime ID of the clrdbg that will be downloaded. Example: ub
 .Parameter InstallPath
 Specifies the path where clrdbg will be installed. Defaults to the directory containing this script.
 
+.Parameter RemoveExistingOnUpgrade
+Remove existing installation directory before upgrade.
+
+.Parameter SkipDownloads
+Skips any download actions.
```
Comment (gregg-miskelly) [inline scripts/GetClrDbg.ps1]: Is there any value in this argument since we don't support the run argument?

File: scripts/GetClrDbg.ps1 [hunk 2]
```diff
@@ -117,22 +169,40 @@ if (-not([System.IO.Path]::IsPathRooted($InstallPath))) {
...
 }
 
-Push-Location $TempPath -ErrorAction Stop
+if ($RemoveExistingOnUpgrade) {
+    if (Test-Path $InstallPath) {
+        Write-Host "Info: $InstallPath exists, deleting." 
+        Remove-Item $InstallPath -Force -Recurse -ErrorAction Stop
+    }
+}
+
```
Comment (gregg-miskelly) [inline scripts/GetClrDbg.ps1]: Don't we need to check for 'IsLatest' before we do this?

**Comment by gregg-miskelly** [comment]
Otherwise LGTM

**Pull request merged**